frontend/test/createProjectForm.test.tsx

```diff
@@ -0,0 +1,55 @@
+import "@testing-library/jest-dom";
+import { render, screen, waitFor } from "@testing-library/react";
+import mockAxios from "jest-mock-axios";
+import { AxiosResponse } from "axios";
+import apiPaths from "../src/properties/apiPaths";
+import CreateProject from "../src/pages/projects/create";
+import {
+    createProjectSubmitHandler,
+    ProjectCreationValues,
+} from "../src/handlers/createProjectSubmitHandler";
+import { getBaseOkResponse } from "./TestEntityProvider";
+
+jest.mock("next/router", () => require("next-router-mock"));
+
+afterEach(() => {
+    mockAxios.reset();
+});
+
+describe("CreateProjectForm initialization", () => {
+    it("Should call axios.get() upon rendering", () => {
+        render(<CreateProject />);
+        expect(mockAxios.get).toHaveBeenCalled();
+
+        expect(screen.getByTestId("projectname-input")).toBeInTheDocument();
+        expect(screen.getByTestId("projectinfo-input")).toBeInTheDocument();
+        expect(screen.getByTestId("versionmanagement-input")).toBeInTheDocument();
+        expect(screen.getByTestId("partnername-input")).toBeInTheDocument();
+        expect(screen.getByTestId("partnerwebsite-input")).toBeInTheDocument();
+        expect(screen.getByTestId("skillinfo-input")).toBeInTheDocument();
+        expect(screen.getByTestId("coach-input")).toBeInTheDocument();
+        expect(screen.getByTestId("skill-input")).toBeInTheDocument();
+    });
+});
+
+it("SubmitHandler for createProject sends post request", async () => {
+    const values: ProjectCreationValues = {
+        projectName: "Test project",
+        projectInfo: "This is a test project",
+        versionManagement: "https://github.com/Test",
+        partnerName: "Test company",
+        partnerWebsite: "testcompany.com",
+        coaches: [],
+        skills: [],
+        skillInfos: [],
+    };
+
+    const ownUserResponse: AxiosResponse = getBaseOkResponse({ _links: { self: { href: "/users/1" } } });
+
+    createProjectSubmitHandler(values);
+    mockAxios.mockResponseFor({ url: apiPaths.ownUser }, ownUserResponse);
+
+    waitFor(() => {
```

Comment:
```suggestion
    await waitFor(() => {
```
Without awaiting a waitFor you might as well remove the test ;)